__tests__/api.test.js

```diff
@@ -29,5 +29,40 @@ describe("/api/topic", () => {
                     expect(typeof data).toBe('object');
                            });
             })
-        })    
+        }) 
+        describe('/api/articles/:article_id', () => {
+            test('GET:200 sends a article to the client', () => {
+              return request(app)
+                .get('/api/articles/1')
+                .expect(200)
+                .then((response) => {
+                    // console.log(response)
+                    expect(response.body.article).toHaveProperty("author", expect.any(String));
+                    expect(response.body.article).toHaveProperty("title", expect.any(String));
+                    expect(response.body.article).toHaveProperty("article_id", expect.any(Number));
+                    expect(response.body.article).toHaveProperty("body", expect.any(String));
+                    expect(response.body.article).toHaveProperty("topic", expect.any(String));
+                    expect(response.body.article).toHaveProperty("created_at", expect.any(String));
+                    expect(response.body.article).toHaveProperty("votes", expect.any(Number));
+                    expect(response.body.article).toHaveProperty("article_img_url", expect.any(String));
```

Comment:
Since you're requesting information on a specific article (with the ID of 1), there's no need to keep this as open ended as 'any String/Number' - you can be specific with your assertions since they're all about an individual article that won't change.

Also as an optional improvement - Look into `toMatchObject`or `objectContaining` as possible alternatives to repeated use of 'toHaveProperty' that will streamline this test a bit